problem-09.c

```diff
@@ -0,0 +1,35 @@
+#include <stdio.h>
+// Prints fibonacci series of a given length
+
+int main()
+{
+    int length, sum, firstNum = 0, secondNum = 1;
+
+    printf("---------FIBONACCI SERIES---------\n");
+
+    printf("Enter the lentgh of fibonacci series: ");
+    scanf("%d", &length);
+
+    if (length == 0)
+    {
+        return 0;
+    }
+
+    for (int i = 0; i < length; i++)
+    {
+        if (i == length - 1)
+        {
+            printf("%d", firstNum);
+        }
+        else
+        {
+            printf("%d, ", firstNum);
+        }

```

Comment:
Better to refactor this as following:

```suggestion
        printf("%d", firstNum);

        if (i != length - 1)
        {
            printf(", ");
        }
```